src/pages/TestBallotDeckScreen.test.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  //electionSample,
+  Election,
+} from '@votingworks/ballot-encoder'
+// TODO: Tally: Use electionSample from ballot-encoder once published.
+
+import React from 'react'
+import { fireEvent } from '@testing-library/react'
+import electionSample from '../data/electionSample.json'
+import { mockOf, render } from '../../test/testUtils'
+import { randomBase64 } from '../utils/random'
+import TestBallotDeckScreen from './TestBallotDeckScreen'
+
+// mock the random value so the snapshots match
+jest.mock('../utils/random')
+const randomBase64Mock = mockOf(randomBase64)
+randomBase64Mock.mockReturnValue('CHhgYxfN5GeqnK8KaVOt1w')
+
+it('renders test decks appropriately', () => {
+  const { container, getByText } = render(
+    <TestBallotDeckScreen
+      appName="VxPrint"
+      appPrecinctId="23"
+      election={electionSample as Election}
+      hideTestDeck={jest.fn()}
+      isLiveMode={false}
+    />
+  )
+
+  fireEvent.click(getByText('All Precincts'))
+
+  expect(container.firstChild).toMatchSnapshot()
```

Comment:
We should not use snapshots when there is data which we can find and verify on the rendered screen. It is too easy for someone to just update the snapshots and assume that everything is fine.

Reply:
I generally agree with you, but in this case I was trying to capture the actual test deck, which I think we may want to have remain stable? (I think I may have captured the wrong snapshot, so I'll review that.)

Does your opinion change if the goal here is to capture the whole test deck and notice if it ever changes?

Reply:
never mind, I found some things I can test in the content that address both issues, so all good. Thanks for the nudge!